python/README.md

```diff
@@ -30,6 +30,18 @@ You can find the list of services below for each container in the group:
 | `idp`        | Base image with [Intel® Distribution for Python]                    |
 | `pip`        | Equivalent python image without [Intel® Distribution for Python]    |
 
+## Run a Performance Sample
+
+To run a performance sample, clone the [Intel® AI Containers](https://github.com/intel/ai-containers) repository, follow the main `README.md` file to setup your environment, and run the following commands:
```

Comment:
we don't expect our target users (who may be novice users) to follow the instructions in the main README of containers repo. Instead of it, we need to provide all the needed commands here (not for building from source, but for pulling and running the container).